frontend/src/components/atoms/hyperTexts/Tag.tsx

```diff
@@ -1,30 +1,73 @@
 import React, { MouseEvent } from 'react';
-import HyperText, { ButtonType } from './HyperText';
+import { Link as ReactRouterLink } from 'react-router-dom';
 
 type Props = {
-  type?: ButtonType;
   icon?: string;
   href?: string;
+  className?: string;
+  small?: boolean;
+  dismiss?: boolean;
+  ariaPressed?: boolean;
   onClick?: (event: MouseEvent<HTMLElement>) => void;
 };
 
 export const Tag: React.FC<Props> = ({
-  type,
-  icon,
+  icon = false,
+  className = '',
   href,
+  small = false,
+  dismiss = false,
+  ariaPressed = false,
   onClick,
   children,
   ...props
-}) => (
-  <HyperText
-    className="fr-tag"
-    type={type}
-    icon={icon}
-    href={href}
-    onClick={onClick}
-    children={children}
-    {...props}
-  />
-);
+}) => {
+  className += ' fr-tag';
+  if (small) {
+    className += ' fr-tag--sm';
+  }
+  if (icon) {
+    className += ` fr-fi-arrow-right-line`;
+  }
+  if (dismiss) {
+    className += ` fr-tag-dismiss`;
+  }
+  if (href) {
```

Comment:
I am concerned with this code being duplicated. We should see these together in a peer programming session.